lib/srv/desktop/windows_server.go

```diff
@@ -858,6 +862,24 @@ func (s *WindowsService) connectRDP(ctx context.Context, log *slog.Logger, tdpCo
 	tdpConn.OnSend = s.makeTDPSendHandler(ctx, recorder, delay, tdpConn, audit)
 	tdpConn.OnRecv = s.makeTDPReceiveHandler(ctx, recorder, delay, tdpConn, audit)
 	width, height := desktop.GetScreenSize()
+
+	computerName, ok := desktop.GetLabel(types.DiscoveryLabelWindowsDNSHostName)
+	if !ok {
+		if computerName, err = utils.Host(desktop.GetAddr()); err != nil {
+			return trace.Wrap(err)
```

Comment:
Can we add a more user-friendly message to this error?

Reply:
Added